test/test_jit.py

```diff
@@ -5230,6 +5275,18 @@ def test_pow_int(x, y):
         self.checkScript(test_pow_float, (2.0, 2.0))
         self.checkScript(test_pow_int, (2.0, 2))
 
+        self.checkScript(test_isnan, (2,))
```

Comment:
Could you get a list of numbers with various properties, and loop through testing them ? negative numbers, 0, etc